s3tests_boto3/functional/test_s3.py

```diff
@@ -1450,6 +1450,22 @@ def test_bucket_list_return_data_versioning():
         assert obj['VersionId'] == key_data['VersionId']
         _compare_dates(obj['LastModified'],key_data['LastModified'])
 
+@pytest.mark.fails_on_dbstore
```

Comment:
i don't think this is needed - there are other `list_object_versions()` tests that work with dbstore